components/script/dom/htmlformelement.rs

```diff
@@ -308,12 +308,17 @@ impl HTMLFormElement {
                         DispositionParam::Filename(Charset::Ext(String::from(charset.clone())),
                                                    None,
                                                    f.name().clone().into()));
+                    /// XXX: unwrap
```

Comment:
Section 4.4, it defaults to text/plain

https://tools.ietf.org/html/rfc7578#section-4.4